rest_framework_gis/fields.py

```diff
@@ -12,21 +12,76 @@
 __all__ = ['GeometryField', 'GeometrySerializerMethodField']
 
 
+def recursive_round(v, precision):
+    """
+    Round all numbers within an array or nested arrays
+      v: number or nested array of numbers
+      precision: integer value of number of decimals to keep
+    """
+    if hasattr(v, '__iter__'):
+        return tuple(recursive_round(x, precision) for x in v)
+    return round(v, precision)
+
+
+def _rm_dupes(coordinates, close=True):
```

Comment:
`_remove_duplicates` would be a better name and more consistent with the rest of naming conventions used in the library.
I'd also make this a static method of `GeometryField`.